hackathon_site/event/views.py

```diff
@@ -83,6 +92,54 @@ def get_context_data(self, **kwargs):
         context["user"] = self.request.user
         context["application"] = getattr(self.request.user, "application", None)
 
+        # Pass in the review and rsvp date information
+        if hasattr(self.request.user, "application") and hasattr(
+            self.request.user.application, "review"
+        ):
+            review = self.request.user.application.review
+
+            context["review"] = review
+            context[
+                "rsvp_passed"
+            ] = _now().date() > review.decision_sent_date + timedelta(
```

Comment:
`decision_sent_date` is optional on the review, and won't be set until we actually send things out. I think that as long as `decision_sent_date` is None, we should not inform them of their application status at all (ie, the `review` context should still be None) as it is still "pending"